website_sale_hide_price/views/res_config_settings_views.xml

```diff
@@ -11,14 +11,17 @@
                     title="Set a default text when hiding prices"
                 >
                     <div class="o_setting_left_pane">
-
+                        <field name="website_hide_price" />
```

Comment:
This is confusing. Put this setting in its own setting box

Reply:
Moved it to its own div.